tests/e2e/service_bindings_test.go

```diff
@@ -169,6 +169,31 @@ var _ = Describe("Service Bindings", func() {
 		})
 	})
 
+	Describe("GET /v3/service_credential_bindings/{guid}/details", func() {
+		var result credentialsResponse
+
+		BeforeEach(func() {
+			brokerGUID := createBroker(serviceBrokerURL)
```

Comment:
This test would be much simpler if we create a binding to a user provided service instance. The details endpoint is valid for both types of service binding.